hazelcast-spring/src/main/resources/hazelcast-spring-3.8.xsd

```diff
@@ -2836,6 +2843,17 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="cluster-data-recovery-policy" type="xs:string" default="FULL_RECOVERY_ONLY">
```

Comment:
Shouldn't this attribute's value be restricted to only valid `xs:enumeration` values?
```
                <xs:attribute name="cluster-data-recovery-policy">
                    <xs:simpleType>
                        <xs:restriction base="non-space-string">
                            <xs:enumeration value="FULL_RECOVERY_ONLY"/>
                            <xs:enumeration value="PARTIAL_RECOVERY_MOST_RECENT"/>
                            <xs:enumeration value="PARTIAL_RECOVERY_MOST_COMPLETE"/>
                        </xs:restriction>
                    </xs:simpleType>
                </xs:attribute>
```